lib/matplotlib/backends/backend_wx.py

```diff
@@ -1614,10 +1614,12 @@ def press(self, event):
             if not self.retinaFix:
                 self.wxoverlay = wx.Overlay()
             else:
-                self.savedRetinaImage = self.canvas.copy_from_bbox(
-                    self.canvas.figure.gca().bbox)
-                self.zoomStartX = event.xdata
-                self.zoomStartY = event.ydata
+                if event.inaxes is not None:
+                    self.savedRetinaImage = self.canvas.copy_from_bbox(
+                        event.inaxes.bbox)
+                    self.zoomStartX = event.xdata
+                    self.zoomStartY = event.ydata
+                    self.zoomAxes = event.inaxes
```

Comment:
No clue if they were public for any particular reason except legacy. We should keep those two as-is, but the zoomAxes should be private under the philosophy that it is easier to make something public later, than to make it private. If it doesn't need to be public, then default to private for anything new.